### PR TITLE
games-misc/lolcat: version bump.

### PR DESCRIPTION
**Pull request by maffblaster**
Package-Manager: Portage-2.3.3, Repoman-2.3.1

### CODE REVIEW

**Comment by monsieurp** [comment]
not too shabby! LGTM, let's ask @chewi what his take on this PR is.

File: games-misc/lolcat/lolcat-42.24.0.ebuild [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
```
Comment (SoapGentoo) [inline games-misc/lolcat/lolcat-42.24.0.ebuild]: - bump to `EAPI=6`
- add all modern ruby impls that work please, and remove old ones

**Comment by chewi** [comment]
Might be good to patch out the need for git. Nothing under dev-ruby seems to need it. The entries in the gemspec that use it don't appear in the version installed by Portage.

Also while I appreciate the description is supposed to be comical, you've made it even more confusing than it was before. I had to visit the site to see what it actually does and even then it's only apparent from a screenshot.

**Comment by maffblaster** [comment]
@chewi I'll make the changes. Turns out ruby-ng.eclass doesn't support EAPI 6 yet, so I'll have to go with 5. I'll also try to bump the package to use newer ruby implementations and remove support for ruby 2.0.

**Comment by mgorny** [comment]
Ping. Is there anything to do here?

**Comment by monsieurp** [comment]
@maffblaster ping?

**Comment by mgorny** [comment]
Closing due to inactivity. Feel free to reopen if there's anything that needs to be done, and you're willing to do it.

**Pull request closed without merging**